src/render/src/encoder.rs

```diff
@@ -59,7 +71,13 @@ impl<R: Resources, C: draw::CommandBuffer<R>> From<C> for Encoder<R, C> {
 }
 
 impl<R: Resources, C: draw::CommandBuffer<R>> Encoder<R, C> {
-    /// Flush the encoded commands onto the device, and clean.
+    /// Submits the commands in this `Encoder`'s internal `CommandBuffer` to the GPU, so they can
+    /// be executed. 
+    /// 
+    /// Calling `flush` before swapping buffers is critical as without it the commands of the
+    /// internal ´CommandBuffer´ will not be sent to the GPU, and as a result they will not be
+    /// processed. Calling flush too often however will result in a performance hit. It is
+    /// generally recommended to call flush once per frame, after every draw call. 
```

Comment:
I meant that as in ` after (every draw call)`. I see how it is ambiguous, fixing it.